packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -306,6 +307,18 @@ const {
 // Selected Rows
 const selectedRows = ref<UsersListQuery['infra']['allUsers']>([]);
 
+const showAdminButton = computed(() =>
+  selectedRows.value.some((user) => !user.isAdmin)
+);
+
+const showRemoveAdminButton = computed(() =>
+  selectedRows.value.some((user) => user.isAdmin)
+);
+
+const showDeleteButton = computed(() =>
+  selectedRows.value.some((user) => !user.isAdmin)
+);
+
```

Comment:
```suggestion
const hasAdminsSelected = computed(() =>
  selectedRows.value.some((user) => !user.isAdmin)
);

const noAdminsSelected = computed(() =>
  selectedRows.value.some((user) => user.isAdmin)
);

```

can we group the computed properties to admin and non-admin selected